packaging/installer/methods/kickstart.md

```diff
@@ -48,33 +48,40 @@ The `kickstart.sh` script does the following after being downloaded and run usin
 The `kickstart.sh` script passes all its parameters to `netdata-installer.sh`, which you can use to customize your
 installation. Here are a few important parameters:
 
--   `--dont-wait`: Enable automated installs by not prompting for permission to install any required packages.
--   `--dont-start-it`: Prevent the installer from starting Netdata automatically.
--   `--stable-channel`: Automatically update only on the release of new major versions.
--   `--nightly-channel`: Automatically update on every new nightly build.
--   `--disable-telemetry`: Opt-out of [anonymous statistics](/docs/anonymous-statistics.md) we use to make
-    Netdata better.
--   `--no-updates`: Prevent automatic updates of any kind.
--   `--reinstall`: If an existing install is detected, reinstall instead of trying to update it. Note that this
-    cannot be used to change installation types.
--   `--local-files`: Used for [offline installations](offline.md). Pass four file paths: the Netdata
+- `--dont-wait`: Synonym for `--non-interactive`
+- `--non-interactive`: Don’t prompt for anything and assume yes whenever possible.
+- `--no-updates`: Disable automatic updates.
+- `--stable-channel`: Use a stable build instead of a nightly build.
+- `--reinstall`: If an existing install is found, reinstall instead of trying to update it in place.
+- `--dont-start-it`: Don’t auto-start the daemon after installing. This will not always work for the new kickstart script, 
+    but is important to keep for some testing scenarios.
```

Comment:
You can drop the info about testing scenarios, and I would just rephrase this as ‘not always guaranteed to work’.